### PR TITLE
PREAPPS-3706: Provide an option to switch to Legacy UI from Zimbra X UI

### PR DESCRIPTION
**Pull request by madhavdhuppe**
Added zimbraPrefClientType attribute into the preferences and preferencesInput.

### CODE REVIEW

File: src/schema/schema.graphql [hunk 1]
```diff
@@ -1289,6 +1289,7 @@ type Preferences {
 	zimbraPrefWebClientOfflineBrowserKey: String
 	zimbraPrefTimeZoneId: String
 	zimbraPrefLocale: String
+	zimbraPrefClientType: String
```
Comment (silentsakky) [inline src/schema/schema.graphql]: should be an enum

Comment (madhavdhuppe) [reply src/schema/schema.graphql]: updated

File: src/schema/schema.graphql [hunk 2]
```diff
@@ -289,6 +289,12 @@ enum ResetPasswordStatus {
 	suspended
 }
 
+enum PrefClientType {
+	advanced,
```
Comment (silentsakky) [inline src/schema/schema.graphql]: this is confusing, we have two options only, right?

Comment (madhavdhuppe) [reply src/schema/schema.graphql]: we are getting three options from API response.

**Pull request merged**